Buildings/Applications/DataCenters/ChillerCooled/Examples/BaseClasses/PostProcess.mo

```diff
@@ -0,0 +1,126 @@
+within Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses;
+partial model PostProcess "Post-processing"
```

Comment:
Is there a reason you have all signals flowing from right to left? This is pretty unconventional and could easily be avoided by adding the post-processing on the right of the system model.